content/rdi/reference/data-transformation-block-types/lookup.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: redis.lookup
+linkTitle: redis.lookup
+description: Get data from another key
+weight: 10
+alwaysopen: false
+categories: ["redis-di"]
+aliases:
+---
+
+Get data from another key
+
+**Properties**
+
+| Name                             | Type           | Description               | Required |
+| -------------------------------- | -------------- | ------------------------- | -------- |
+| [**connection**](#option1fields) | `string`       | Connection name<br/>      | yes      |
+| [**cmd**](#option1fields)        | `string`       | Command to execute<br/>   | yes      |
+| [**args**](#option1fields)       | `list[string]` | Command's parameters<br/> | yes      |
+| [**language**](#option1fields)   | `string`       | Expression language<br/>  | yes      |
+| [**field**](#option1fields)      | `string`       | Target field<br/>         | yes      |
+
+**Additional Properties:** not allowed
+
```

Comment:
Add list of allowed commands (even if we are not validating these can only be commands that do read of one or more keys / members
Also we should document how different commands response will look like in a field